### PR TITLE
More clown uplink only items

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Dear god why
Clown HONK can now get a DNA injector that has the Clumsy gene
Honk, Clown angets can now get HONK PINS
All Surpluse = 0 in role restricted removed - Code improvement  
[why]
Clown pins are for clowns to use other traitor guns or lock out others form their own guns
Clown DNA are for clowns to work with a team if they are one, having their DNA injectors and such
All role items get 0 surpluse do to it being a child of the role restriction so no need to add in 6 more lines of code that dose nothing

### CODE REVIEW

**Comment by IrkallaEpsilon** [comment]
Interesting idea, are the injectors instant? Is it fixable by genetics?

**Comment by Trilbyspaceclone** [comment]
@IrkallaEpsilon Injectors are not instant and genetics cant fit it

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1355,13 +1355,28 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/reverse_bear_trap
 	restricted_roles = list("Clown")
 
+/datum/uplink_item/role_restricted/clumsyDNA
+	name = "Clumsy Clown DNA"
+	desc = "A DNA injector that has been loaded with the clown gene that makes people clumsy. \
```
Comment (Ignari-contrib) [inline code/modules/uplink/uplink_items.dm]: desc = "A DNA injector that has been loaded with the clown gene that makes people clumsy.

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -1355,13 +1355,28 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	Making someone clumsy will allow them to use clown firing pins as well as Reverse Revolvers. For a laugh try useing this on HOS to see how many times they shoot themselfs in the foot!"
```
Comment (Ignari-contrib) [inline code/modules/uplink/uplink_items.dm]: Making someone clumsy will allow them to use clown firing pins as well as Reverse Revolvers. For a laugh try using this on the HOS to see how many times they shoot themselves in the foot!"

File: code/modules/uplink/uplink_items.dm [hunk 3]
```diff
@@ -1355,13 +1355,28 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	cost = 3
+	item = /obj/item/dnainjector/clumsymut
+	restricted_roles = list("Clown")
+
+/datum/uplink_item/role_restricted/anitclumsyDNA
+	name = "Clown punishment"
+	desc = "HONK Looks like one of are angents got, got! \
```
Comment (Ignari-contrib) [inline code/modules/uplink/uplink_items.dm]: desc = "HONK! Looks like one of our agents got got!

File: code/modules/uplink/uplink_items.dm [hunk 4]
```diff
@@ -1355,13 +1355,28 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	We got a barrle of laughs for you, HONK, we got a Mime DNA injector HONK! Use this on a clown HONK to make them not clusmy HONK!"
```
Comment (Ignari-contrib) [inline code/modules/uplink/uplink_items.dm]: We got a barrel of laughs for you, HONK, we got a Mime DNA injector HONK! Use this on a clown HONK to make them not clumsy HONK!"

File: code/modules/uplink/uplink_items.dm [hunk 5]
```diff
@@ -1355,13 +1355,28 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	cost = 1 //If Hos or warden gets a uplink they should be able to get it for allmost free
```
Comment (Ignari-contrib) [inline code/modules/uplink/uplink_items.dm]: cost = 1 //If Hos or warden gets an uplink they should be able to get it for almost free

**Review by silicons — CHANGES_REQUESTED** [review]
restricted items should be niche useful items and I fail to see the legitimate use of these DNA injectors.

**Comment by Trilbyspaceclone** [comment]
@kevinz000  
CLOWNUT trait lets them use their own firing pins
Buy sketchin, remove pin, add your own, Bam you can now use it without shooting your foot
*or*
Stun/KO sec, inject them, watch them now shoot their foot

**Review by silicons — CHANGES_REQUESTED** [review]
The clown punishment thing is just silly and doesn't need to be an uplink item
clumsy clown injector can stand to be 1-2 TC
Also, get rid of the undocumented changes of getting rid of surplus = 0, you should know better by now.

**Review by silicons — APPROVED** [review]

**Pull request merged**